docs: Update docs for v6.7.0 (#5593)
plotly/plotly.py: doc/python/colorscales.md

### Explicitly Constructing a Color scale

> The Plotly Express `color_continuous_scale` argument accepts explicitly-constructed color scales as well.  There is an equivalent graph objects example using the `colorscale` parameter [below](#custom-heatmap-color-scale-with-graph-objects).

Reviewer: ```suggestion
The Plotly Express `color_continuous_scale` argument accepts explicitly-constructed color scales as well.  

The Graph Objects `colorscale` parameter also accepts an explicitly-constructed color scale, as shown in the example [below](#custom-heatmap-color-scale-with-graph-objects).
```

wording suggestion, since the two arguments aren't _quite_ equivalent (one accepts a list of strings while the other accepts a list of `[number, string]` lists)

Author: Good catch.
